client/src/redux/fetch-user-saga.js

```diff
@@ -19,6 +19,13 @@ function* fetchSessionUser() {
       data: { user = {}, result = '', sessionMeta = {} }
     } = yield call(getSessionUser);
     const appUser = user[result] || {};
+
+    const [userId] = Object.keys(user);
+
+    const sound = user[userId].sound;
+
+    localStorage.setItem('fcc-sound', sound);
```

Comment:
Should we be using `store` here instead?

Reply:
I don't understand what additional benefits `store` provide over `localStorage` api, also syntactically both are similar.

Reply:
I believe `store` will adapt to use other storage APIs if the user is using a non-modern browser.